tests/libtest/test_rhv.py

```diff
@@ -0,0 +1,66 @@
+import logging
+import time
+
+from ocs_ci.deployment.rhv import RHVIPI
+from ocs_ci.framework.testlib import libtest
+from ocs_ci.framework.pytest_customization.marks import rhv_platform_required
+from ocs_ci.ocs.node import get_node_objs
+from ocs_ci.ocs.platform_nodes import RHVNodes
+
+logger = logging.getLogger(__name__)
+
+
+@libtest
+@rhv_platform_required
+def test_get_vm_status():
+    """
+    Test of RHV get_vm_status() method implementation
+    VM  of healthy OCS Cluster has 'up' status by default.
+    """
+    rhv_depl = RHVIPI()
+    vm_name = get_node_objs()[0].get().get("metadata").get("name")
+    logger.info(f"vm names is: {vm_name}")
```

Comment:
This does not run on multiple nodes, correct? If not maybe better to log here `vm name` rather than `vm names`

Reply:
in-cooperated the change